hasher-matcher-actioner/webapp/src/Api.tsx

```diff
@@ -299,19 +300,22 @@ type Dataset = {
   in_use: boolean;
   hash_count: number;
   match_count: number;
+  pdqMatchThreshold?: string;
 };
 
 export async function updateDataset(
   privacyGroupId: string,
   fetcherActive: boolean,
   writeBack: boolean,
   matcherActive: boolean,
+  pdqMatchThreshold?: string,
```

Comment:
Why not Number?

Reply:
because of the key distinction between "" and 0 for this value. Having this be a string for the purposes of the UI makes validation cleaner